memtier_benchmark.cpp

```diff
@@ -1070,9 +1070,29 @@ int main(int argc, char *argv[])
             fprintf(stderr, "error: use no-expiry only with data-import\n");
             exit(1);
         }
-        
         obj_gen = new object_generator();
         assert(obj_gen != NULL);
+    }
+    else if (cfg.crc_verify) {
+        if (cfg.data_verify) {
+            fprintf(stderr, "error: use data-verify only with data-import\n");
+            exit(1);
+        }
+        if (cfg.no_expiry) {
+            fprintf(stderr, "error: use no-expiry only with data-import\n");
```

Comment:
can you please explain this error message too?

Reply:
see above